.gitignore

```diff
@@ -1,15 +1,18 @@
+*.cache1
+*.cache2
```

Comment:
What about somewhat more descriptive names? `.cache-pre-dev`, `.cache-pre-inst`? I think this would be more feature-proof, in case different stages are added.

I don't grok the meaning of `-f`. I'd expect that `-i` means that the cached images will be used. Why do you need to specify `-f` in addition? Also, `-ff` negates `-f`, which is counter-intuitive.

Looking at the bigger picture, it seems that the incremental builds should become the default (just because they're much more pleasant). We have the following (orthogonal) choices: 1. use cached images if available or force regeneration, 2. leave cached images around for the next run or not, 3. remove cached images or let them be. I think `--incremental` shoudl control option 1., not sure about the rest. But `--force` seems to be a bad fit for controlling 2. and 3.